lab-david/route/photo-router.js

```diff
@@ -0,0 +1,99 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const del = require('del');
+const AWS = require('aws-sdk');
+const multer = require('multer');
+const Router = require('express').Router;
+const createError = require('http-errors');
+const debug = require('debug')('cfgram:photo-router');
+
+const Photo = require('../model/photo.js');
+const PhotoAlbum = require('../model/photoalbum.js');
+const bearerAuth = require('../lib/bearer-auth-middleware.js');
+const photoRouter = module.exports = Router();
+
+AWS.config.setPromisesDependency(require('bluebird'));
```

Comment:
Interesting, haven't seen this before. Way to find that. s3 actually has promises built in. You can use them by adding `.promise()` to the end of your s3 call.